src/main/webapp/gast/einzelbeleg.jsp

```diff
@@ -5,325 +5,372 @@
-  <jsp:param name="Formular" value="einzelbeleg"/>
-  <jsp:param name="Datenfeld" value="ID"/>
-  <jsp:param name="size" value="11"/>
+    <jsp:param name="ID" value="<%= id%>"/>
+    <jsp:param name="Formular" value="einzelbeleg"/>
+    <jsp:param name="Datenfeld" value="ID"/>
+    <jsp:param name="size" value="11"/>
 </jsp:include>
 
 <!----------ID---------->
-<div id="id">
-  <jsp:include page="../forms/id.jsp">
-    <jsp:param name="ID" value="<%=id%>"/>
-    <jsp:param name="title" value="gast_einzelbeleg"/>
-  </jsp:include>
+<div class="container" id="id">
+    <jsp:include page="../forms/id.jsp">
+        <jsp:param name="ID" value="<%=id%>"/>
+        <jsp:param name="title" value="gast_einzelbeleg"/>
+    </jsp:include>
 </div>
 
 <!----------Belegstelle---------->
-<h3><% Language.printTextfield(out, session, "einzelbeleg", "TabBelegstelle"); %></h3>
+<h3 class="ut-heading ut-heading--h3"><% Language.printTextfield(out, session, "einzelbeleg", "TabBelegstelle"); %></h3>
 
-<table class="content-table">
-  <tbody>
-    <tr>
-      <th><% Language.printDatafield(out, session, "einzelbeleg", "PersonRO"); %></th>
-      <td>
-        <jsp:include page="../inc.erzeugeFormular.jsp">
-          <jsp:param name="ID" value="<%= id %>"/>
-          <jsp:param name="Formular" value="einzelbeleg"/>
-          <jsp:param name="Datenfeld" value="PersonRO"/>
-          <jsp:param name="Readonly" value="yes"/>
-        </jsp:include>
-      </td>
-    </tr>
-    <tr>
-      <th><% Language.printDatafield(out, session, "einzelbeleg", "Belegform"); %></th>
-      <td>
-        <jsp:include page="../inc.erzeugeFormular.jsp">
-          <jsp:param name="ID" value="<%= id %>"/>
-          <jsp:param name="Formular" value="einzelbeleg"/>
-          <jsp:param name="Datenfeld" value="Belegform"/>
-          <jsp:param name="size" value="50"/>
-          <jsp:param name="Readonly" value="yes"/>
-        </jsp:include>
-        <jsp:include page="../inc.erzeugeFormular.jsp">
-          <jsp:param name="ID" value="<%= id %>"/>
-          <jsp:param name="Formular" value="einzelbeleg"/>
-          <jsp:param name="Datenfeld" value="Griechisch"/>
-          <jsp:param name="size" value="50"/>
-          <jsp:param name="Readonly" value="yes"/>
-        </jsp:include>
-      </td>
-    </tr>
-    <tr>
-      <th><% Language.printDatafield(out, session, "einzelbeleg", "LemmaRO"); %></th>
-      <td>
-        <jsp:include page="../inc.erzeugeFormular.jsp">
-          <jsp:param name="ID" value="<%= id %>"/>
-          <jsp:param name="Formular" value="einzelbeleg"/>
-          <jsp:param name="Datenfeld" value="LemmaRO"/>
-          <jsp:param name="Readonly" value="yes"/>
-        </jsp:include>
-      </td>
-    </tr>
+<table class="ut-table ut-table--striped ut-table--striped--color-primary-3">
+    <tbody class="ut-table__body">
+        <tr class="ut-table__row">
+            <th class="ut-table__item ut-table__header__item" scope="col">
```

Comment:
Using a `th` and a `td` with the given ut-... classes in the same `tr` leads to design problems. I am not sure whether the original university design provides a good solution for this. There are examples where e.g. the first column in the table has a different font color or background color, but i doubt that would look good. 

There is a class for the table element that you can add as example:
`<table class="ut-table ut-table__first-col--color-primary-1">`

This should lead to the font of the first column being displayed as red. Maybe we can define a similar custom class on our own CSS file and use that? (Instead of changing the color to red we could simply make the font bold).

From the naming scheme, it could look e.g. like this:
`<table class="ut-table ut-table--striped nppm-table--striped--color-bold">`

But, as mentioned, it is not allowed to have th inside tbody, so maybe you need to replace the ths with regular tds for this to look the same as in the design guide.

Reply:
done